style.css

```diff
@@ -0,0 +1,97 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: sans-serif;
+}
+body {
+  display: flex;
+
+  font-size: 30px;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+  background: #fbfbfb;
+  color: #cacaca;
+}
+h1 {
```

Comment:
I think if you change h1 color than entry-label class then it would be more readable.........